organization/_posts/2021/2021-09-02-coordination.md

```diff
@@ -0,0 +1,149 @@
+---
+title: "HSF Coordination Meeting #213, 2 September 2021"
+layout: plain_toc
+---
+
+Present: Graeme Stewart, Josh McFayden, Attila Krasznahorkay, Kyle Knoepfel, Michel Jouvin, Pere Mato, Ben Morgan, Dorothea vom Bruch, Krzysztof Genser, Mark Neubauer, Caterina Doglioni, Eduardo Rodrigues, Allison Hall, Philippe Gras, Liz Sexton-Kennedy, Alexander Moreno, Andrei Gheata, Gordon Watts, Jerry Ling, Michael Hernandez, Nicola Skidmore, Sudhir Malik, Torre Wenaus, Daniel Elvira, Benedikt Hegner, Aman Goel
```

Comment:
```suggestion
Present: Graeme Stewart, Josh McFayden, Attila Krasznahorkay, Kyle Knoepfel, Michel Jouvin, Pere Mato, Ben Morgan, Dorothea vom Bruch, Krzysztof Genser, Mark Neubauer, Caterina Doglioni, Eduardo Rodrigues, Allison Hall, Philippe Gras, Liz Sexton-Kennedy, Alexander Moreno, Andrei Gheata, Gordon Watts, Jerry Ling, Michael Hernandez, Nicole Skidmore, Sudhir Malik, Torre Wenaus, Daniel Elvira, Benedikt Hegner, Aman Goel
```

Reply:
"Nicole" is how the gets everyone to call her, hence this update (her ID name is Nicola indeed).